api/lib/reducers/cosmosV2-reducers.js

```diff
@@ -241,7 +241,7 @@ function proposalReducer(
     networkId,
     id: Number(proposal.id),
     proposalId: String(proposal.id),
-    type: proposalTypeEnumDictionary[proposal.content.type],
+    type: proposalTypeEnumDictionary[proposal.content.type.split('/')[1]],
```

Comment:
why is this better?

Reply:
because Terra has different names for the proposal types. Look at the change in the dictionary.

Instead of `cosmos-sdk/ParametersChanges` it is called `params/ParametersChanges`, for example